code/game/machinery/vending/marine_vending.dm

```diff
@@ -177,6 +177,7 @@
 	wrenchable = FALSE
 	products = list(
 					/obj/item/storage/large_holster/machete/full = 10,
+					/obj/item/storage/large_holster/katana/full = 10,
```

Comment:
Oh, I forgot to say, this is overriden on cm_initialize.dm, because the vending product amounts scale with player pop. Go modify that file too.